workers/unity/Packages/manifest.json

```diff
@@ -4,10 +4,10 @@
     "com.unity.entities": "0.0.12-preview.8",
     "com.unity.incrementalcompiler": "0.0.42-preview.14",
     "com.unity.package-manager-ui": "1.9.9",
+    "com.unity.modules.audio": "1.0.0",
     "com.unity.modules.imgui": "1.0.0",
```

Comment:
can we get rid of imgui and ui as well? think it was also a dependency

Reply:
All of these are for the playground game